pkg/apis/pxc/v1/pxc_backup_types.go

```diff
@@ -53,6 +53,10 @@ type PXCBackupSpec struct {
 	ActiveDeadlineSeconds    *int64                  `json:"activeDeadlineSeconds,omitempty"`
 	StartingDeadlineSeconds  *int64                  `json:"startingDeadlineSeconds,omitempty"`
 	SuspendedDeadlineSeconds *int64                  `json:"suspendedDeadlineSeconds,omitempty"`
+	// RunningDeadlineSeconds is the number of seconds to wait for the backup to transition to the 'Running' state.
+	// Once this threshold is reached, the backup will be marked as failed.
+	// When unspecified, uses the value from the parent cluster's .spec.backup.runningDeadlineSeconds (which defaults to 5m).
+	RunningDeadlineSeconds *int32 `json:"runningDeadlineSeconds,omitempty"`
```

Comment:
Do we have a reason why we used `int32` instead of `int64` which is what we already use for `activeDeadlineSeconds` and `suspendedDeadlineSeconds`?

Reply:
I'm fine with switching to `int64` for consistency, but on the other hand `int64` seems like overkill (`int32` is large enough as it is to represent an equivalent of 68 years)

Reply:
I agree with you on the rational, but I think it is better to keep them consistent since they are covering the same area.